nuget.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<packageSources>
+		<add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+		<add key="MyGet" value="https://dotnet.myget.org/F/uwpcommunitytoolkit/api/v3/index.json" protocolVersion="3" />
```

Comment:
👀👀👀
This is a timing issue with the publishing of the sample app and the nugets, but make sure to remove this whenever the new version of the sample app is public, as well as the nugets.